cpp/src/grpc/codegen/generate_conversions.py

```
@@ -0,0 +1,2662 @@
+        if edef:
+            from_fn = _enum_from_proto_fn(ftype, edef)
+            lines.append(
+                f"{ind}cpu_problem.{setter}({from_fn}(pb_problem.{pname}()));"
+            )
+        else:
+            lines.append(f"{ind}cpu_problem.{setter}(pb_problem.{pname}());")
+
+    lines.append("")
+
+    # Setter groups — guard on first field having data
+    for gname, gdef in setter_groups.items():
```

Ruff (0.15.12)

[warning] Loop control variable `gname` not used within loop body

Rename unused `gname` to `_gname`

(B007)